docs/admin/logs/_index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Logging"
+weight: 50
+type: "docs"
+aliases:
+  - /docs/install/collecting-logs
```

Comment:
Do we want to make "logs" a top-level next to "install", add a "configuration" sub-tree to hold all the configuration, or have a "common" sub-tree to hold configuration that's common to Serving and Eventing (off the top of my head, the high-availability configuration, logging, and monitoring)?

(I'm trying to avoid the "too many items at one level in the LHS navigation" problem we have today.)

Reply:
Can you add this as a suggestion to the Miro doc please so I can see better what you're saying / how this fits in?
We can follow it up in another PR if we want to move this

Reply:
@evankanderson tracing too?